File: java/fibonacciseries.java
import java.util.Scanner;

public class fibonacciseries
{
    static int fibonacci(int n)
    {
        if(n==0 || n==1)
        {
            return n;
        }
        return fibonacci(n-1)+fibonacci(n-2);
    }
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n =sc.nextInt();
        int i=0;
        while(i<n)
        {
            System.out.println(fibonacci(i));
            i++;
        }
    }
}

File: python/fibonacciseries.py
def fibonacci(n):
    if n==0 or n==1:
        return n
    else:
        return fibonacci(n-1)+fibonacci(n-2)

if __name__ == "__main__":
    n=int(input())
    i=0
    while i<n:
        print(fibonacci(i))
        i+=1